### DISCUSSION TITLE
How can I create an ERD for a imported SQL script?

### DISCUSSION THREAD

**Discussion by davehouser1** [dbeaver/dbeaver#20877] [Q&A] [answered]
Hello I want to create a Visualization / ERD for an imported SQL script, is this possible?
I cannot connect directly to our database with DBeaver host. I can only export the Postgres dump file as SQL. Is it possible to build a ERD with this file in DBeaver?

**Comment by E1izabeth** [comment, marked as answer]
I believe that our [Visual Query Builder](https://dbeaver.com/docs/wiki/Visual-Query-Builder/) might help you, but this feature is included only to our PRO products. You could try a 30-days trial version for free to test if it fits your case https://dbeaver.com/download/
